### DISCUSSION TITLE
Adaptation of toclevel parameters

### DISCUSSION THREAD

**Discussion by Auggie321** [gethinode/hinode#556] [Help] [answered]
Under the current theme, is there any consideration for adding the adaptation of the toclevel parameter?

**Comment by markdumay** [comment, marked as answer]
You can configure the table of contents in Hugo's [markup configuration](https://gohugo.io/getting-started/configuration-markup/#table-of-contents). Hinode uses these settings to define the heading levels to include. Is that what you're looking for?

**Comment by markdumay** [comment]
Additionally, you can enable/disable the TOC inclusion in Hinode's [navigation configuration](https://gethinode.com/docs/configuration/navigation/#basic-configuration).

**Reply by Auggie321** [reply]
sorry, I don't know where to add toclevel information in your template, see the two links you provided, and compare them.
It has been added to markup.toml to successfully display multiple levels.

`markup.toml`
```
    ## The first two lines have been modified to remove the background color
    # style = "monokailight" 
******
[tableOfContents]
    endLevel = 5
    ordered = false
    startLevel = 2

[goldmark]
    [goldmark.extensions]
        definitionList = true
        footnote = true
******
```
It would be nice to add this tableOfContents parameter to your examplesite configuration.
Thank you for your help.

**Comment by Auggie321** [comment]
resolved
